### PR TITLE
Fix crash during loading if default charset is ascii

### PR DESCRIPTION
**Pull request by c-goldschmidt**
Should fix https://github.com/serversigns/ServerSigns/issues/14

### CODE REVIEW

**Comment by EpiCanard** [comment]
Thanks for the PR, I will make more tests tomorrow to verify that everything works as expected

File: src/main/java/de/czymm/serversigns/translations/MessageHandler.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ private void copyEmbedded(String resourceName, Path target) {
                 Files.deleteIfExists(target);
                 Files.createFile(target);
 
-                try (BufferedWriter writer = Files.newBufferedWriter(target, Charset.defaultCharset()); BufferedReader reader = new BufferedReader(new InputStreamReader(resource))) {
+                try (BufferedWriter writer = Files.newBufferedWriter(target, Charset.forName('UTF-8')); BufferedReader reader = new BufferedReader(new InputStreamReader(resource))) {
```
Comment (EpiCanard) [inline src/main/java/de/czymm/serversigns/translations/MessageHandler.java]: There is also a use of **defaultCharset** inside **ConfigGenerator** and **AdminListener**. Can you change them ?

Comment (EpiCanard) [reply src/main/java/de/czymm/serversigns/translations/MessageHandler.java]: You can use `StandardCharsets.UTF_8` instead of `Charset.forName`

**Review by EpiCanard — CHANGES_REQUESTED** [review]

File: src/main/java/de/czymm/serversigns/translations/MessageHandler.java (hunk 1 above)
Comment (c-goldschmidt) [reply src/main/java/de/czymm/serversigns/translations/MessageHandler.java]: done

**Review by EpiCanard — APPROVED** [review]
Perfect

**Pull request merged**